common/src/main/scala/uk/ac/wellcome/models/aws/SQSMessage.scala

```diff
@@ -2,6 +2,7 @@ package uk.ac.wellcome.models.aws
 
 import io.circe.generic.extras.JsonKey
 
+// Deprecated in favour of uk.ac.wellcome.sns.NotificationMessage
```

Comment:
Better yet here is a (scala deprecated annotation)[https://www.scala-lang.org/api/2.11.8/index.html#scala.deprecated]. This will cause IntelliJ to winge every time we use this class